cypress/e2e/QueueVisualizer.cy.ts

```diff
@@ -0,0 +1,64 @@
+import uaLocalization from "../../src/assets/i18n/ua.json";
+
+const QUEUE_TEST_MSG = "Test item";
+const PATH_NAME_QUEUE = "/queue";
+const QUEUE_LANG_MATCHER =
+  uaLocalization.translation.queueVisualizer.buttons.add;
+
+const inputTextAddNewQueueItem = () => {
+  cy.getByTestId("data-list-text-input").type(QUEUE_TEST_MSG);
+  cy.getByTestId("data-list-add-button").click();
+  cy.getByTestId("queue-list-items-container")
+    .getByTestId("data-list-item")
+    .contains(QUEUE_TEST_MSG)
+    .should("exist");
+  cy.getByTestId("queue-list-items-container")
+    .getByTestId("data-list-item")
+    .should("have.length", 4);
+};
+
+const removeQueueItem = () => {
+  cy.getByTestId("data-list-remove-button").click();
+  cy.getByTestId("queue-list-items-container")
+    .getByTestId("data-list-item")
+    .contains("1")
+    .should("not.exist");
+  cy.getByTestId("queue-list-items-container")
+    .getByTestId("data-list-item")
+    .should("have.length", 2);
+};
+
+const processQueueBackButton = () => {
+  cy.getByTestId("data-list-back-button").click();
+  cy.location("pathname").should("eq", "/");
+};
+
+const processQueueSelectLanguage = () => {
+  cy.getByTestId(`language-selector-body`).click();
+  cy.getByTestId(`sentinelStart`).next().children().get("ul li").last().click();
```

Comment:
Selector is very risky, are there any other alternatives? If it's a normal selector for material ui library, please create a separated selector for it since you are working with the third party library selector.